lib/manager/gradle/index.ts

```diff
@@ -25,50 +27,59 @@ import { platform } from '../../platform';
 import { LANGUAGE_JAVA } from '../../constants/languages';
 import * as datasourceMaven from '../../datasource/maven';
 import { DatasourceError } from '../../datasource';
+import { BinarySource } from '../../util/exec/common';
```

Comment:
Can you cleanup the import please. Remove the empty lines and reorder imports. eslint should help.

Reply:
I can remove the empty lines, but what is the issue with the ordering? The only rule `eslint` enforces is to have external modules before internal modules.

Would you like them sorted alphabetically?

Reply:
Removing empty lines and move external packages to top is enough

Reply:
External packages _were_ already on top. I removed the empty lines and moved the modules from the same directory to the bottom.